packages/vendor-core/package.json

```diff
@@ -24,13 +24,13 @@
   "dependencies": {
     "@apollo/client": "^3.3.7",
     "@novnc/novnc": "novnc/noVNC#25b3d49d322b0a7c9ee1e071d93042d70f5176b7 ",
-    "@patternfly/patternfly": "^4.70.2",
-    "@patternfly/react-catalog-view-extension": "^4.8.126",
-    "@patternfly/react-core": "^4.84.4",
-    "@patternfly/react-icons": "^4.7.22",
-    "@patternfly/react-styles": "^4.7.22",
-    "@patternfly/react-table": "^4.19.45",
-    "@patternfly/react-tokens": "^4.9.22",
+    "@patternfly/patternfly": "^4.65.5",
```

Comment:
@MariaAga should this be lower than the previous version?

Reply:
Nope, bad rebase, removed